iceoryx_hoofs/include/iceoryx_hoofs/cxx/type_traits.hpp

```diff
@@ -28,6 +28,7 @@ namespace iox
 {
 namespace cxx
 {
+// AXIVION DISABLE STYLE AutosarC++19_03-A18.1.1 : C-array type usage is intentional
```

Comment:
It seems the C-array is only used at one place at `is_char_array`. If that is the case, could you add the suppression only there and also use that justification?

Reply:
@elBoberido you're right this is taken care.Thanks